sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/EventGridPublisherAsyncClient.java

```diff
@@ -70,17 +74,26 @@ public EventGridServiceVersion getServiceVersion() {
      * @return the completion.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Void> sendEvents(Iterable<EventGridEvent> events) {
+    public Mono<Void> sendEventGridEvents(Iterable<EventGridEvent> events) {
```

Comment:
document `@throws` for `NullPointerException`  .

Reply:
Added